base/strings/util.jl

```diff
@@ -482,6 +482,29 @@ julia> replace("The quick foxes run quickly.", r"fox(es)?" => s"bus\\1")
 replace(s::AbstractString, pat_f::Pair; count=typemax(Int)) =
     replace(String(s), pat_f, count=count)
 
+
+"""
+    replace(s::AbstractString, reps::Pair...)
+
+Perform a successive series of `replace` operations on `s`, evaluating the replacements in `reps...`
+sequentially from left to right.
+
+# Examples
+```jldoctest
+julia> replace("foobar", "foo"=>"bar", "bar"=>"baz")
+"bazbaz"
+
+julia> replace("foobar","bar"=>"baz", "foo"=>"bar")
+"barbaz"
+!!! note
+    This method is not currently recommended for performance critical code; consider a regular
+    expression based approach in such cases instead.
+```
```

Comment:
The note should be put outside of the code block

Reply:
Whoops! Thanks.